data/linux-bridge/0004-bridge-marker-daemonset.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: bridge-marker
+  namespace: kube-system
```

Comment:
Please put it in the same namespace as the bridge CNI.

Reply:
Done